INSTALL.md

```diff
@@ -49,20 +61,24 @@ sudo apt-get install -y libbpf-dev
 ### Install libbpf from source
 
 ```bash
-https_proxy="https://user:password@proxy.server:port" git clone https://github.com/libbpf/libbpf.git
+git clone https://github.com/libbpf/libbpf.git
 cd libbpf
-git checkout ...
-make -C src
-sudo make -C src install
+git checkout v0.5.0  # or v0.6.1 if needing a newer version
+make -C src && sudo make -C src install
 ```
 
 The library and pkgconfig file is installed to /usr/lib64, which is not where the loader or
-pkg-config look. Fix this by editing the ldconfig and exporting the PKG_CONFIG_PATH.
+pkg-config looks. Fix this by editing the ldconfig file as suggested below.
 
 ```bash
 sudo vim /etc/ld.so.conf.d/x86_64-linux-gnu.conf
-# add /usr/lib64 line to the bottom of the file, save and exit.
+# add a line with /usr/lib64 to the bottom of the file, save and exit.
 sudo ldconfig
+```
+
+The following export statment may not be required, but it does not hurt.
```

Comment:
typo => s/statment/statement

Suggest:
```
The following statement may be necessary if libbpf is installed from source instead of the package manager.
```